### PR TITLE
[14.0][ADD] product_assortment_details module

### PR DESCRIPTION
**Pull request by ilyasProgrammer**

### CODE REVIEW

**Review by francesco-ooops — APPROVED** [review]
LGTM

**Review by elvise — APPROVED** [review]
LGTM !!!

**Comment by ilyasProgrammer** [comment]
@OCA/product-maintainers

**Comment by francesco-ooops** [comment]
@rousseldenis @JordiBForgeFlow what do you say?

File: product_assortment_details/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Product Assortment Details",
```
Comment (rousseldenis) [inline product_assortment_details/__manifest__.py]: Details is a little bit too vague.

As you add a description, should be product_assortment_description

Comment (francesco-ooops) [reply product_assortment_details/__manifest__.py]: sure, we can change that. Went for "details" as it's description + chatter.

**Comment by francesco-ooops** [comment]
@OCA/product-maintainers any chance for merge?

**Comment by GSLabIt** [comment]
@ilyasProgrammer could you please rebase?

**Review by rousseldenis — APPROVED** [review]

**Comment by francesco-ooops** [comment]
@rousseldenis can we also merge? :)

**Pull request merged**